### PR TITLE
Change log and release blog for 1.3.2

### PR DESCRIPTION
**Pull request by TimMoore**

### CODE REVIEW

File: src/blog/lagom-1-3-2.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+Because of this unexpected inconsistency between the development and production environments, Lagom 1.3.2 deprecates the development environment behavior and prints a warning if these properties are not defined in a service that uses Cassandra persistence. The `cassandraKeyspace` parameter for the Lagom Maven plugin and `lagomCassandraKeyspace` setting for the Lagom sbt plugin are also deprecated, as it was not clear to everyone that these settings only affect the development environment, and are not used when keyspaces are configured in `application.conf`.
+
+New projects built from the Lagom Maven archetype or sbt giter8 templates include this configuration by default, using the name of the project it prompts you for to assign a default keyspace to the example persistent service implementation. You can change this default as required, and should be sure to configure unique keyspaces for additional service implementations as you create them.
```
Comment (ignasi35) [inline src/blog/lagom-1-3-2.md]: I think "sbt giter8 templates" should be replaced with "Giter8 templates".

Comment (TimMoore) [inline src/blog/lagom-1-3-2.md]: I'll change the capitalization.

I do think it should say "sbt" somewhere... I'm not sure that everyone reading will know what Giter8 is. I've seen a lot of users refer to them as "sbt templates" on Gitter, issues, etc.

How about "Giter8 templates for sbt" instead?

Comment (ignasi35) [inline src/blog/lagom-1-3-2.md]: How about:
"New projects built from the Lagom Maven archetype or using `sbt new` and Lagom's g8 templates (either [java](https://github.com/lagom/lagom-java.g8) or [scala](https://github.com/lagom/lagom-scala.g8) ) include this configuration by default ... "

Comment (ignasi35) [inline src/blog/lagom-1-3-2.md]: the problem I have with "Giter 8" is that we don't mention "Giter 8" anywhere and I think the most probable bit our users will remember is `sbt new` and, maybe, some `.g8`in the URL.

Comment (TimMoore) [inline src/blog/lagom-1-3-2.md]: OK 😄 sounds like we agree with each other. I'll use your second suggestion.

**Pull request merged**